releases/v4.16.0.md

```diff
@@ -47,6 +41,48 @@ numeral16 : "0" [xX] ("_"* hex_char+)+
 float     : numeral10 "." numeral10? [eE[+-]numeral10]
 ```
 
+### Additional new featues
+
+* [#6300](https://github.com/leanprover/lean4/pull/6300) adds the `debug.proofAsSorry` option. When enabled, the proofs
+of theorems are ignored and replaced with `sorry`.
+
+* [#6362](https://github.com/leanprover/lean4/pull/6362) adds the `--error=kind` option (shorthand: `-Ekind`) to the
+`lean` CLI. When set, messages of `kind` (e.g.,
+`linter.unusedVariables`) will be reported as errors. This setting does
+nothing in interactive contexts (e.g., the server).
+
+* [#6366](https://github.com/leanprover/lean4/pull/6366) adds support for `Float32` and fixes a bug in the runtime.
+
+### Library updates
+
+The Lean 4 library saw many changes that improve arithmetic reasoning, enhance data structure APIs,
+and refine library organization. Key changes include better support for bitwise operations, shifts,
+and conversions, expanded lemmas for `Array`, `Vector`, and `List`, and improved ordering definitions.
+Some modules have been reorganized for clarity, and internal refinements ensure greater consistency and correctness.
+
+### Breaking changes
+
+[#6330](https://github.com/leanprover/lean4/pull/6330) removes unnecessary parameters from the funcion induction
+principles. This is a breaking change; broken code can typically be adjusted
+simply by passing fewer parameters.
+
+_This highlights section was contributed by Violetta Sim._
```

Comment:
Should this be a link to a website or profile?

Reply:
Wow, thank you for pointing out the typos, I am surprised that I missed them myself

As for the web page, I don't have one at the moment, maybe github profile could be used instead?
Btw, I don't mind if I am credited or not, I'm fine with both (just happy to contribute), and it was only a reorganization of the release notes entries.

So would you prefer to leave it as is, use my github profile, or remove the credit altogether?

Reply:
That's your call to make. I think you should be mentioned. If you don't have a web presence you want to promote then it's fine as is.